packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -249,6 +249,54 @@ class Transforms {
             }));
         }
 
+        // Decrypt content url, web url and display name if extension is present
+        if(activity.extension){
+
+          //Decrypt content url for extension custom app
+          if(activity.extension.objectType =='extension' && activity.extension.extensionType =='customApp'){
+            promises.push(requestWithRetries(ctx.webex.internal.encryption, ctx.webex.internal.encryption.decryptText,
+             [activity.encryptionKeyUrl, activity.extension.contentUrl, {onBehalfOf: container.onBehalfOfUser}])
+             .then((decryptedContentUrl) => {
+                activity.extension.contentUrl = decryptedContentUrl;
+             })
+            .catch((reason) => {
+              ctx.webex.logger.error(`Decrypt activity.extension.contentUrl error for activity ${activity.activityId} in container ${activity.targetId}: ${reason}`);
+              activity.error = reason;
+              return Promise.resolve(object);
+            }));
+
+          }
+
+          // Decrypt web url for extension custom app
```

Comment:
for phase 1, we could remove web url addition in the response. This was decided by Sheshadri and Kalyana

Reply:
Removed web URL attribute.